### PR TITLE
AAU, on Github, when I look at the kili-playground, I see it is renamed kili-python-sdk

### PR DESCRIPTION
**Pull request by theodu**
- [ ] I opened a merge request on `kili` on a branch with the same name than the branch of the current pull request and forced security tests.
- [ ] I opened a pull request on the Node playground if the changes are breaking.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,15 +17,16 @@ Kili is a platform that empowers a data-centric approach to Machine Learning thr
 
 and [many more](https://cloud.kili-technology.com/docs/labeling/labeling-overview/#available-labeling-job-types-per-asset-type-a-idjob-types-per-asset-typea).
 
-## What is Kili Playground?
-
-Kili Playground is the home of the **Kili Python SDK**. It comes with several [recipes](recipes/) that demonstrate its most frequent use cases.
+## What is Kili Python SDK?
 
+**Kili Python SDK** is the Software Development Kit of Kili. It allows to automate actions on the platform, query and mutate data and provide a better experience. It comes with several [recipes](recipes/) that demonstrate its most frequent use cases.
```
Comment (PierreLeveau) [inline README.md:22]: **Kili Python SDK** is the Python client for the Kili platform. It allows to query and manipulate the main entities available in Kili, like projects, assets, labels, api keys... It comes with several [recipes](recipes/) that demonstrate how to use it in the most frequent use cases.

File: README.md [hunk 2]
```diff
@@ -40,17 +42,23 @@ pip install kili
...
 from kili.client import Kili
 kili = Kili()
 # You can now use the Kili client!
 ```
+
 Note that you can also pass the API key as an argument of the `Kili` initialization:
+
 ```python
-kili = Kili(api_key='MY API KEY')
+kili = Kili(api_key='<you api key value here>')
```
Comment (PierreLeveau) [inline README.md:55]: your

File: recipes/README.md [hunk 3]
```diff
@@ -41,31 +41,34 @@ python delete_all_assets.py
 
 ## How to push pre-annotation to existing assets
 
-Check out [our recipe on how to import predictions](https://github.com/kili-technology/kili-playground/blob/master/recipes/import_predictions.ipynb) (run it [here](https://colab.research.google.com/github/kili-technology/kili-playground/blob/master/recipes/import_predictions.ipynb))
+Check out [our recipe on how to import predictions](https://github.com/kili-technology/kili-python-sdk/blob/master/recipes/import_predictions.ipynb) (run it [here](https://colab.research.google.com/github/kili-technology/kili-python-sdk/blob/master/recipes/import_predictions.ipynb))
```
Comment (PierreLeveau) [inline recipes/README.md:44]: here and below, relative paths should work too

**Review by PierreLeveau — CHANGES_REQUESTED** [review]

File: recipes/README.md (hunk 3 above)
Comment (theodu) [reply recipes/README.md:44]: won't they work when the repo will be renamed?

**Review by marcenacp — APPROVED** [review]

**Review by PierreLeveau — APPROVED** [review]

**Pull request merged**